libs/examples/src/table/table-expandable-rows-example/table-expandable-rows-example.html

```diff
@@ -1,4 +1,8 @@
-<dt-table [dataSource]="dataSource" [multiExpand]="multiExpand">
+<dt-table
+  [dataSource]="dataSource"
+  [multiExpand]="multiExpand"
+  [showExportButton]="true"
```

Comment:
Please don't add export here. Either create a new example for exporting + expandable row or update one of the export stories to have the expandable rows